index.d.ts

```diff
@@ -1,3 +1,6 @@
 export default class FloatingFocus {
-	constructor(container: Element = document.body) {}
+	constructor(
+		container: Element = document.body,
+		opts: Partial<{ noInlineStyle: boolean }> = {},
+	) {}
```

Comment:
I'd actually like this to just have just 1 argument being the options, with `container` being another field on that... but then it's a breaking change which feels meh ;_;

Reply:
Ah that works for me (: